lib/team_result_parser.rb

```diff
@@ -1,8 +1,14 @@
+# This class will pull from game_teams.csv.
+# We need game_id, team_id, HoA, result, head_coach, goals, shots, tackles
+
```

Comment:
Adding this description is super helpful for understanding what we will be using the `team_result_parser` for!